### PR TITLE
Feat : Added cart-page

### PR DESCRIPTION
**Pull request by sushil-mod**
Added Cart page 
Minor Changes in header links
Added footer icons

### CODE REVIEW

File: screens/html/cart.html [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+            </div>
+
+        </div>
+
+
+
+    </div>
+
+
```
Comment (kotalwar-krishna) [inline screens/html/cart.html:211]: remove unnecessary spaces

**Review by kotalwar-krishna — APPROVED** [review]
remove unnecessary spaces

**Review by Megha-Pathak — APPROVED** [review]

**Comment by Megha-Pathak** [comment]
Looks good to merge, just format the code once and remove extra line spaces.

File: screens/html/cart.html (hunk 1 above)
Comment (sushil-mod) [reply screens/html/cart.html:211]: ok, thanks

**Pull request merged**